src/layouts/partials/footer.html

```diff
@@ -4,7 +4,7 @@
 
     <div id="vintage-disclaimer">
       <div id="vintage-disclaimer-inner">
-        <div id="vintage-disclaimer-wording"><p>This is part of our documentation corresponds to our vintage product. Currently we are working actively to provide up-to-date documentation on the latest generation of our product, using the Kubernetes Cluster API.</p></div>
+        <div id="vintage-disclaimer-wording"><p>This is part of our documentation may be no accurate since we are migrating out from our vintage product. Currently we are working actively to provide up-to-date documentation on the latest generation of our product, using the Kubernetes Cluster API.</p></div>
```

Comment:
```suggestion
        <div id="vintage-disclaimer-wording"><p>This is part of our documentation may not be accurate because we are migrating away from our vintage product. Currently we are working actively to provide up-to-date documentation on the latest generation of our product, using the Kubernetes Cluster API.</p></div>
```